src/support/components/FeedbackQuestionsOverlay.tsx

```diff
@@ -114,4 +123,11 @@ const FeedbackQuestionsOverlay: FC<OwnProps> = () => {
   )
 }
 
-export default FeedbackQuestionsOverlay
+const mdtp = {
+  showOverlay,
+  dismissOverlay,
+}
+
+const connector = connect(null, mdtp)
+
+export default connector(FeedbackQuestionsOverlay)
```

Comment:
we can skip all this by using the `useDispatch` hook from `react-redux`. 

[here's](https://github.com/influxdata/ui/blob/master/src/cloud/components/RateLimitAlert.tsx#L68) an example of `useDispatch` being used in combination with `showOverlay`